CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # AppSignal for Node.js Changelog
 
+## 3.0.0
+
+Welcome to AppSignal for Node.js version 3.0.0.!
+
+In this release, we have switched to using OpenTelemetry internally for tracing. (We do not yet support OpenTelemetry for metrics and logging.) It is no longer needed to use AppSignal instrumentation packages like `@appsignal/express`, `@appsignal/koa`, `@appsignal/nextjs`, `@appsignal/apollo-server`, etc. Some official OpenTelemetry instrumentation packages are bundled with our package as dependencies and instrumented automatically. Other _supported instrumentation packages_ can be [added and configured manually](https://docs.appsignal.com/nodejs/3.x/instrumentation/additional-instrumentations.html).
+
+We strongly recommend following the [migration guide] while upgrading. Instrumentation from package version 2 is not compatible with version 3. Custom instrumentation will need to be set up using [OpenTelemetry instrumentation](https://docs.appsignal.com/nodejs/3.x/instrumentation/instrumentation.html).
+
+If you run into any issues upgrading to this new version, please contact us at [support@appsignal.com][support] for help.
+
+- [Version 3 migration guide][migration guide]
+- [Contact us for support][support]
+
+[migration guide]: https://docs.appsignal.com/nodejs/3.x/migration-guide.html
+[support]: mailto:support@appsignal.com
+
+### Added
```

Comment:
We also support [Next.js](https://docs.appsignal.com/nodejs/3.x/integrations/nextjs.html)